packages/mephisto-review-hook/src/index.js

```diff
@@ -1,54 +1,64 @@
-import React, { useState, useEffect } from "react";
+import React, { useState, useEffect, useCallback } from "react";
+import { useMephistoReview as useMephistoReviewLegacy } from "./legacy";
 
-function useMephistoReview({ useMock, mock } = {}) {
+function useMephistoReview({
+  useMock,
+  mock,
+  taskId,
+  page,
+  resultsPerPage,
+} = {}) {
   const [data, setData] = useState(null);
-  const [counter, setCounter] = useState(0);
+  const [isLoading, setIsLoading] = useState(false);
   const [error, setError] = useState(null);
 
   const isMock =
     mock !== undefined && (useMock === undefined || useMock === true);
 
   useEffect(() => {
-    if (isMock) {
-      return;
-    }
-
-    fetch("/data_for_current_task")
-      .catch((err) => {
-        setError({ type: "DATA_RETRIEVAL", error: err });
-      })
+    if (isMock) return;
```

Comment:
So hooks can't have early returns. See #389 for how I fixed this. You'll want to incorporate those changes here, otherwise it will revert the behavior.

Reply:
whoops, i misread this - my bad!